project/AppDependencies.scala

```diff
@@ -19,7 +19,8 @@ object AppDependencies {
     "uk.gov.hmrc" %% "play-language" % "3.4.0",
     "com.thoughtworks.xstream" % "xstream" % "1.4.10",
     "uk.gov.hmrc" %% "wco-dec" % "0.23.0",
-    "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
+    "ai.x"         %% "play-json-extensions" % "0.9.0",
```

Comment:
In my opinion we shouldn't use this and remove whole jackson dependencies.

Reply:
the library allows has to simplify son formats , I tried to not use and it is not that easy to get away from wco-dec as of now